awx/ui_next/src/components/ListHeader/ListHeader.test.jsx

```diff
@@ -71,7 +71,7 @@ describe('ListHeader', () => {
     expect(history.location.search).toEqual(query);
     const toolbar = wrapper.find('DataListToolbar');
     toolbar.prop('clearAllFilters')();
-    expect(history.location.search).toEqual('?item.page_size=5');
+    expect(history.location.search).toEqual('');
```

Comment:
this was testing that custom page size would not be stripped out when selecting the clear all button...the fact it needed to be changed does lead me to believe there may be a bug with the refactor?